internal/engine.go

```diff
@@ -220,6 +227,26 @@ func (e *Engine) cleanupTemp(temp string) {
 	}
 }
 
+func (e *Engine) filterIgnoredPaths(issues []tt.Issue) []tt.Issue {
+	filtered := make([]tt.Issue, 0, len(issues))
+	for _, issue := range issues {
+		if !e.isIgnoredPath(issue.Filename) {
+			filtered = append(filtered, issue)
+		}
+	}
+	return filtered
+}
+
+func (e *Engine) isIgnoredPath(path string) bool {
+	for _, ignored := range e.ignoredPaths {
+		if strings.HasPrefix(path, ignored) {
```

Comment:
When using `strings.HasPrefix` for path matching, unintended behaviour might happen. For 
example, if you want to ignore `/foo`, `/foobar` might also be ignored.

How about using `filepath.Match` instead? Plus, this case must be test out.